### PR TITLE
Styling/movie summary

### PR DESCRIPTION
**Pull request by michaelputnam67**
## What does this PR do?
Add styling elements
## Where should the reviewer start?
App.js

## How should this be manually tested?
Run npm test

### CODE REVIEW

**Review by sted1994 — APPROVED** [review]
Cool that you got the formatter working. Good Job

File: src/Movie-Summary.js [hunk 1]
```diff
@@ -21,10 +19,34 @@ const MovieSummary = ({movie}) => {
 					<p>{movie.movieData.average_rating}</p>
 					<p>{movie.movieData.tagline}</p>
 				</div>
-				<ul> Statistics:
-					<li>{movie.movieData.budget}</li>
-					<li>{movie.movieData.revenue}</li>
-					<li>Run Time: {movie.movieData.runtime}</li>
+				<ul className='statistics'>
```
Comment (emmacbd) [inline src/Movie-Summary.js]: thanks for breaking these up into NumberFormat components, looks much better!

File: src/css/movie-summary.css [hunk 2]
```diff
@@ -5,21 +5,35 @@
 	grid-template-columns: .10fr 1fr .10fr;
 }
 
+.about { 
+	color: white;
+}
+
+.movie-summary {
```
Comment (emmacbd) [inline src/css/movie-summary.css:8]: like this new styling, much more readable/easier to uniformly render across movies.

**Review by emmacbd — COMMENTED** [review]
Looking much better, thanks Michael

**Pull request merged**